nilearn/maskers/base_masker.py

```diff
@@ -424,3 +429,105 @@ def __sklearn_tags__(self):
             surf_img=True, niimg_like=False, masker=True
         )
         return tags
+
+    def transform(self, imgs, confounds=None, sample_mask=None):
+        """Apply mask, spatial and temporal preprocessing.
+
+        Parameters
+        ----------
+        imgs : 1D/2D Surface image
+            Images to process.
```

Comment:
`imgs` can be 1D/2D Surface image or list of images, non?

Reply:
yup